tbx/project_styleguide/templates/patterns/pages/blog/blog-posting-jsonld.html

```diff
@@ -0,0 +1,29 @@
+{% load wagtailcore_tags wagtailimages_tags %}
```

Comment:
This template shows the limits of using templates to generate JSON. It seems very fragile to me:
* Everything has to use `|escapejs`
* tags like `{% if ... %}` need to go on the line above where they'd normally go to avoid extra blank lines
* Similarly you have to be careful about trailing commas.

It would be a lot more robust if we were to turn this into a template tag that would build the right `dict` in Python then use `json.dumps` to generate valid JSON. What do you think?